haproxy.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: haproxy
+  version: 2.6.7
+  epoch: 0
+  description: "A TCP/HTTP reverse proxy for high availability environments"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+      - "*"
+      attestation: TODO
+      license: (GPL-2.0-or-later AND GPL-2.1-or-later) WITH OpenSSL-Exception
+  dependencies:
+    runtime:
+      - libgcc
+  scriptlets:
+    pre-install: |
+      #!/bin/sh
+      addgroup -S haproxy 2>/dev/null
+      adduser -S -D -H -h /var/lib/haproxy -s /bin/false \
+        -G haproxy -g haproxy haproxy 2>/dev/null
+
+      # move config to new location
+      if [ -e /etc/haproxy.cfg ] && ! [ -e /etc/haproxy/haproxy.cfg ]; then
+        mkdir -p /etc/haproxy
+        mv /etc/haproxy.cfg /etc/haproxy/haproxy.cfg
+      fi
+
+      exit 0
+    pre-upgrade: |
+      #!/bin/sh
+      addgroup -S haproxy 2>/dev/null
```

Comment:
I think we can probably delete these scriptlets... all they seem to do is handle the group creation and stuff.

Reply:
@kaniini Is there a way (or will there be a way) in Melange to designate users/groups to install with the given package?